test/unit/range/views/view_minimiser_test.cpp

```diff
@@ -46,21 +61,41 @@ struct iterator_fixture<iterator_type> : public ::testing::Test
     seqan3::views::minimiser(vec, 5);
 };
 
-using test_type = ::testing::Types<iterator_type>;
-INSTANTIATE_TYPED_TEST_SUITE_P(iterator_fixture, iterator_fixture, test_type, );
+template <>
+struct iterator_fixture<two_ranges_iterator_type> : public ::testing::Test
+{
+    using iterator_tag = std::forward_iterator_tag;
+    static constexpr bool const_iterable = false;
+
+    seqan3::dna4_vector text{"ACGGCGACGTTTAG"_dna4};
+    decltype(seqan3::views::kmer_hash(text, seqan3::ungapped{4})) vec = text | kmer_view;
+    result_t expected_range{26, 97, 27, 6, 1};
+
+    decltype(seqan3::views::minimiser(seqan3::views::kmer_hash(text, seqan3::ungapped{4}), 5, text | rev_kmer_view))
+    test_range = seqan3::views::minimiser(vec, 5, text | rev_kmer_view);
+
+};
+
+
+using test_types = ::testing::Types<iterator_type, two_ranges_iterator_type>;
+INSTANTIATE_TYPED_TEST_SUITE_P(iterator_fixture, iterator_fixture, test_types, );
 
 template <typename T>
 class minimiser_view_properties_test: public ::testing::Test { };
 
 using underlying_range_types = ::testing::Types<std::vector<seqan3::dna4>,
                                                 std::vector<seqan3::dna4> const,
+#if SEQAN3_WORKAROUND_ISSUE_1743
+                                                // seqan3::bitcompressed_vector<seqan3::dna4>,
+                                                // seqan3::bitcompressed_vector<seqan3::dna4> const,
```

Comment:
why is this commented out if it is wrapped inside of a workaround. @seqan/core I thought we don't want to use workarounds inside of the test because it is not a workaround?

Reply:
I don't know. @marehr added this. :see_no_evil:

Reply:
@marehr can you explain?